pkg/apis/eksctl.io/v1alpha5/defaults.go

```diff
@@ -1,5 +1,15 @@
 package v1alpha5
 
+// SetClusterConfigDefaults will set defaults for a given cluster
+func SetClusterConfigDefaults(cfg *ClusterConfig) {
+	if cfg.HasClusterCloudWatchLogging() && len(cfg.CloudWatch.ClusterLogging.EnableTypes) == 1 {
+		switch cfg.CloudWatch.ClusterLogging.EnableTypes[0] {
+		case "all", "*":
+			cfg.CloudWatch.ClusterLogging.EnableTypes = SupportedCloudWatchClusterLogTypes()
```

Comment:
I guess removing and combining would be a little more correct?

Reply:
Sorry, I didn't get it. Could you please elaborate?